src/main/java/net/comments/objects/BCommentManagerDuplicate.java

```diff
@@ -1,6 +1,15 @@
 package net.comments.objects;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
 public class BCommentManagerDuplicate implements CommentManager {
```

Comment:
Why do you use a separate class for implementing each interface methods? Please implement 3 methods in the one class.